packages/auth/src/accessToken/service.ts

```diff
@@ -65,3 +75,27 @@ async function introspect(
     return { active: true, ...grant, key: { proof: 'httpsig', jwk: keys[0] } }
   }
 }
+
+async function revoke(
```

Comment:
Could you add tests for that in `service.test.ts`?

Reply:
added